### ISSUE TITLE
require-v-for-key false positive

### ISSUE THREAD

**Issue by askovpen** [vuejs/eslint-plugin-vue#785]
```javascript
<option v-for="(value, key) in list.a_cls" v-bind:value="value">{{ key }}</option>
```
error: Elements in iteration expect to have 'v-bind:key' directives (vue/require-v-for-key)

**Comment by LinusBorg** [comment]
This is not a false positive. there's not `v-bind:key` on your <option> object.
